misc/etl_bot.sh.in

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 # Simple script to start ET: Legacy client/listen server with Omni-Bots
 #
-./etl@BIN_SUFFIX@ +set omnibot_enable 1 +set omnibot_path "./legacy/omni-bot"
+./etl@BIN_SUFFIX@ +set omnibot_enable 1 +set omnibot_path "./legacy/omni-bot" $*
```

Comment:
You might want to use `"$@"` instead of `$*` to avoid unwanted word splitting.

```suggestion
./etl@BIN_SUFFIX@ +set omnibot_enable 1 +set omnibot_path "./legacy/omni-bot" "$@"
```